requirements/base.txt

```diff
@@ -3,4 +3,4 @@ jsonschema<5,>=3.2  # TODO: evaluate risk of removing jsonschema 3.x support
 typing_extensions>=4.4,<5 # 3.7 doesn't have Literal
 
 # resource validation & schema generation
-pydantic~=1.8
+pydantic>=1.8
```

Comment:
Is there any specific reason to use `>=1.8` instead of `2.1`?

Reply:
There is a call for keeping pydantic 1.x support: https://github.com/aws/serverless-application-model/pull/3249#issuecomment-1634028897